python/ray/services.py

```diff
@@ -1315,6 +1365,7 @@ def start_ray_processes(address_info=None,
                         num_redis_shards=1,
                         redis_max_clients=None,
                         redis_protected_mode=False,
+                        redis_password=None,
```

Comment:
We probably want to remove all instances of the `redis_protected_mode`/`protected_mode` argument, right? And also essentially remove everything from https://github.com/ray-project/ray/pull/2697, right (e.g., the code for making the temp file and stuff)?

Reply:
A follow up PR is fine.